crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -279,7 +373,7 @@ async fn app_validation_workflow_inner(
     let awaiting_ops = awaiting_ops.load(Ordering::SeqCst);
     let rejected_ops = rejected_ops.load(Ordering::SeqCst);
     let ops_validated = accepted_ops + rejected_ops;
-    tracing::debug!("{ops_validated} out of {num_ops_to_validate} validated: {accepted_ops} accepted, {awaiting_ops} awaiting deps, {rejected_ops} rejected.");
+    tracing::info!("{ops_validated} out of {num_ops_to_validate} validated: {accepted_ops} accepted, {awaiting_ops} awaiting deps, {rejected_ops} rejected.");
```

Comment:
This seems like debug level to me, did you change this intentionally?

Reply:
I did, based on my debugging efforts lately with the workflows. If I enable logs for this module, I'd like to see the summary of a validation run. When put at debug which is the same level as other logs higher up in that function, I get "such and such number of ops will be validated" followed by "ops filtered out because hashes are missing" and then this which I found noisy.

I don't have a clear sense of which level to use and if we're considering the levels when tracing is enabled globally or per module.